migrate.sh

```diff
@@ -24,6 +32,7 @@ invenio communities custom-fields init
 python -m zenodo_rdm.migrator site/zenodo_rdm/migrator/streams.yaml
 
 # Update ID sequences in DB
+# TODO this is being developed programatically. Might not be needed at all.
```

Comment:
I think we can remove this since there is the alter sequence PR. Or at least fully comment it.